test/fixtures/bpmn/non-regression/lanes.with.sublanes.bpmn

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
:question: why not adding the sublanes to the existing pool diagram
for instance, add a new lane with a label that has 2 sublanes: 1st one with label, the 2nd one without label.
This will avoid an extra test + test the label rendering in sublanes

Reply:
I was thinking to have them in a separate file as after we will add here the vertical lanes

Reply:
we already have a vertical pool file that could be updated when we will implement vertical lanes

Reply:
ok will move the lanes there